tests/js/web/FilterBarComponentTest.js

```diff
@@ -508,5 +508,9 @@ describe('FilterBarComponent', function () {
 		it('should add a label to the filterbar for select inputs', function () {
 			expect(this.$container.find('span[data-filter-id="size"]')).to.have.length(1);
 		});
+
+		it('should hide the add filter button if all filters have been used', function () {
```

Comment:
Might be missing something but why is this passing? There's 4 inputs in the test DOM and only three of them have values when this test is run.

Reply:
There's an odd bug here where the colour select2 is being treated as if there's a value, this also happens when I take the test fieldset into a Lexicon UI to test...